tools/localizations/cn.csv

```diff
@@ -72,6 +72,8 @@ Aqara Wireless Mini Switch T1,Aqara 无线开关 T1
 "LeTianPai Smart Switch 2 Way",乐天派智能开关（零火版）双开
 "LeTianPai Smart Switch 3 Way",乐天派智能开关（零火版）三开
 "LeTianPai Smart Light Bulb",乐天派智能灯泡
+"Heiman Door and Window D1-M",智能门磁传感器
+"Heiman Motion Sensor",智能人体带光照度探测器
```

Comment:
@Detailzc I dont believe we need the line for the "Heiman Motion Sensor" as I see it currently here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/6e4ec75be413951ce8127d6c4dbb71dfd442ce40/tools/localizations/cn.csv#L9